.github/workflows/pull_request.yml

```diff
@@ -9,22 +9,15 @@ jobs:
         name: Soundness
         uses: apple/swift-nio/.github/workflows/soundness.yml@main
         with:
-            api_breakage_check_enabled: true
-            broken_symlink_check_enabled: true
-            docs_check_enabled: true
-            format_check_enabled: true
-            license_header_check_enabled: true
             license_header_check_project_name: "SwiftOpenAPIGenerator"
-            shell_check_enabled: true
-            unacceptable_language_check_enabled: true
 
```

Comment:
Why are we removing these?

Reply:
I assume because they're on by default?

Reply:
Correct. The default is `true` so no reason to pass it additionally

Reply:
Heh, OK. We discussed why we wanted them in when we onboarded them in the earlier PR, but fine.